### DISCUSSION TITLE
systemd and flatpak

### DISCUSSION THREAD

**Discussion by Yujkore** [cockpit-project/cockpit#18827] [General]
So, I'm new to cockpit, and I want to be able to daily-drive VM's instead of having to go through a whole installation process every-time i want to distro-hop.

Well anyway, I ran an archinstall script and noticed that i didn't have an option to select systemd bootloader like what i'm used to using. I went ahead and moved forward with a grub installation and it completed just fine. I now notice that flatpaks take forever to load, but native applications otherwise load normally. IDK if this has anything to do with the bootloader i have to use but I've never had this problem when running an archinstall script on physical hardware.

Is there a reason why systemd is no longer an option for me?

Is there some other reason why my flatpaks are slow to load?

Thanks in advance!

**Comment by martinpitt** [comment]
Neither selecting arch bootloaders nor slow flatpak loading are related to Cockpit in any way. For the latter I suggest watching the journal, sometimes there are useful error messages. If they take around 25 seconds, it's usually a D-Bus timeout, i.e. trying to call a service which does not exist or crashes.
